src/server/utilities/liteATITracking/clickTracking/index.ts

```diff
@@ -20,7 +22,11 @@ export default () => {
       const anchorElement = targetElement as HTMLAnchorElement;
       const nextPageUrl = anchorElement?.href;
 
-      window.processClientDeviceAndSendLite(atiURL as string);
+      if (atiURL && !clicktrackingFiredUrls.includes(atiURL)) {
```

Comment:
We don't need a null check for atiURL here - it will always have a value
```suggestion
      if (!clicktrackingFiredUrls.includes(atiURL)) {
```

Reply:
Heya, `.getAttribute('data-lite-ati-click')` has type undefined|string, could we possibly keep it as a fail safe?

Reply:
Can do.